server/base/src/main/java/org/apache/accumulo/server/manager/state/TabletManagementIterator.java

```diff
@@ -279,6 +279,11 @@ private boolean shouldReturnDueToLocation(final TabletMetadata tm,
             || (tm.getHostingGoal() == TabletHostingGoal.ONDEMAND && tm.getHostingRequested()))) {
           return true;
         }
+        // If the Tablet has walogs and operation id then need to return so
+        // TGW can bring online to process the logs
+        if (tm.getLogs().isEmpty() && tm.getOperationId() != null) {
```

Comment:
May want to limit this to merging operation for now.  Split will need this once its fully implement (#3844).  Delete table will never need it.

```suggestion
        if (tm.getLogs().isEmpty() && tm.getOperationId() != null && tm.getOperationId().getType() == TabletOperationType.MERGING) {
```